src/CustomConnectors/CcoDashboardAzureConnector/CcoDashboardAzureConnector.pq

```diff
@@ -251,7 +251,7 @@ shared GetPoliciesTable = (govKind as text) as table =>
 // Retrieve all policies for a subscription
 Policies = (govKind, subscriptionId, token) =>
     let
-        url = GetManagementUri(govKind, "subscriptions/"&subscriptionId&"/providers/Microsoft.PolicyInsights/policyStates/default/queryResults?api-version=2019-10-01"),
```

Comment:
for the approach without the middleware we want to keep the latest status (default includes all the changes on status):
https://docs.microsoft.com/en-us/rest/api/policy/policy-states/list-query-results-for-resource#policystatesresource